html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationProvisionings.js

```diff
@@ -276,6 +292,44 @@ export const pfConfigurationProvisioningFields = {
       ]
     }
   },
+  cert_chain: ({ options: { meta = {} } } = {}) => {
+    return {
+      label: i18n.t('The certificate chain for the signer certificate'),
+      text: i18n.t('The certificate chain of the signer certificate in pem format.'),
+      fields: [
+        {
+          key: 'cert_chain',
+          component: pfFormTextarea,
+          attrs: {
+            ...pfConfigurationAttributesFromMeta(meta, 'cert_chain'),
+            ...{
+              rows: 5
+            }
+          },
+          validators: pfConfigurationValidatorsFromMeta(meta, 'cert_chain', 'Chain')
```

Comment:
`'Chain'` must be localized.

Reply:
It is localized here https://github.com/inverse-inc/packetfence/blob/devel/html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js#L221

and here https://github.com/inverse-inc/packetfence/blob/devel/html/pfappserver/root/static.alt/src/globals/configuration/pfConfiguration.js#L225

Reply:
I know, but it doesn't work when extracting the localizable strings with our script. Each string to be localized must be surrounded by a i18n function.